builtin/settingtypes.txt

```diff
@@ -699,6 +704,11 @@ meshgen_block_cache_size (Mapblock mesh generator's MapBlock cache size MB) int
 #    Enables minimap.
 enable_minimap (Minimap) bool true
 
+#    Enables fullbright mode.
+#    This allows you to see in the dark if you have the required fullbright
+#    privilege.
+enable_fullbright_mode (Fullbright mode) bool false
```

Comment:
'enable' is a redundant word that is unfortunately used for some other settings, please remove it.
fullbright mode = true / false is enough to make sense.